psi4/driver/driver_nbody.py

```diff
@@ -336,12 +340,13 @@ def nbody_gufunc(func, method_string, **kwargs):
         nocp_ptype_body_dict = {n: np.zeros(arr_shape) for n in nbody_range}
         vmfc_ptype_body_dict = {n: np.zeros(arr_shape) for n in nbody_range}
     else:
-        cp_ptype_by_level, cp_ptype_body_dict = None, None
-        nocp_ptype_by_level, nocp_ptype_body_dict = None, None
-        vmfc_ptype_body_dict = None
+        cp_ptype_by_level, cp_ptype_body_dict = {}, {}
```

Comment:
For posterity's sake, the reason I had used the `NoneType` check in the final var setting loop [here](https://github.com/psi4/psi4/pull/952/files#diff-8348f203a0eebe7b8ff993f7e6b17c43R474) is that these dicts were originally initialized to `None`, then set to the variable-containing dicts of interest [here](https://github.com/psi4/psi4/pull/952/files#diff-8348f203a0eebe7b8ff993f7e6b17c43R444).